src/Form/NewArticleForm.php

```diff
@@ -8,27 +8,41 @@
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\Form\FormEvent;
 use Symfony\Component\Form\Extension\Core\Type\DateType;
 use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
 use Symfony\Component\Form\Extension\Core\Type\TextareaType;
 use Symfony\Component\Form\Extension\Core\Type\SubmitType;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use App\Repository\UserRepository;
 
 class NewArticleForm extends AbstractType
```

Comment:
@JanMaria Teraz połączymy formularze tworzenia i edycji w jeden. Te dwie klasy usuń i utwórz nową `ArticleType`. Jako, że dotychczas determinowaliśmy opcje pola isPublished w dwojaki sposób będziesz musiał to obsłużyć i zaimplementować logikę sprawdzającą czy formularz jest nowym czy edytującym. Nie rób tak, że dodasz nową opcję isEdit i w kontrolerze przekażesz raz true raz false. Chcę, żebyś zdeterminował z jakim formularzem masz aktualnie do czynienia sprawdzając stan obiektu encji Article zbindowanej do formularza.